File: src/main/java/com/plog/global/exception/exceptions/HashTagException.java
package com.plog.global.exception.exceptions;

import com.plog.global.exception.errorCode.ErrorCode;
import com.plog.global.exception.errorCode.HashTagErrorCode;

public class HashTagException extends BaseException{

    public HashTagException(HashTagErrorCode errorCode) {
        super(errorCode);
    }

    public HashTagException(HashTagErrorCode errorCode, String logMessage) {
        super(errorCode, logMessage);
    }

    public HashTagException(HashTagErrorCode errorCode, String logMessage, String clientMessage) {
        super(errorCode, logMessage, clientMessage);
    }

}

File: src/main/kotlin/com/plog/global/exception/exceptions/HashTagException.kt
package com.plog.global.exception.exceptions

import com.plog.global.exception.errorCode.HashTagErrorCode

class HashTagException : BaseException {
    constructor(errorCode: HashTagErrorCode) : super(errorCode)
    constructor(errorCode: HashTagErrorCode, logMessage: String) : super(errorCode, logMessage)
    constructor(errorCode: HashTagErrorCode, logMessage: String, clientMessage: String) : super(errorCode, logMessage, clientMessage)
}
